packages/components/src/components/hds/dropdown/toggle/icon.js

```diff
@@ -78,9 +78,11 @@ export default class HdsDropdownToggleIconComponent extends Component {
    */
   get iconSize() {
     if (this.args.size === 'small') {
```

Comment:
maybe something like this would be more clear? 🤔
```suggestion
    if (this.args.size === 'medium' && !this.hasChevron) {
      // in this special case we use a larger SVG
      return '24';
    } else {
      // this is the default size (notice: for the "small" variant with chevron, we set the actual size to `12px` via CSS)
      return '16';
    }
```